### ISSUE TITLE
Why softmax_crossentropy in MultitaskClassifier

### ISSUE THREAD

**Issue by Marcosuff** [deepchem/deepchem#1608]
Hello,
This might be a silly question but why does the softmax+crossentropy is used in MultitaskClassifier? For instance, if a compound is labeled active or 1 in many tasks, isnt a sigmoid+crossentropy loss function the  way to output independent probabilities?

**Comment by peastman** [comment]
MultitaskClassifier outputs a distribution over n classes for each task.  Those output probabilities are computed with a softmax function, hence the loss function.

**Comment by Marcosuff** [comment]
> MultitaskClassifier outputs a distribution over n classes for each task. Those output probabilities are computed with a softmax function, hence the loss function.

I see. So there tasks are independent. I'm new to deep learning and MT and was curious because the output layer has as many nodes as there are tasks, so I thought the output probabilities would sum to 1. Thanks for clarifying @peastman!
